__docs__/wonder-blocks-labeled-field/labeled-field.stories.tsx

```diff
@@ -265,86 +262,76 @@ const AllFields = (
                         }
                         instantValidation={false}
                         disabled={disabled}
-                        readOnly={readOnly}
                     />
                 }
             />
 
-            {/* SingleSelect does not support readonly state */}
```

Comment:
Decided to include SingleSelect and MultiSelect for the readonly state so we can make sure all the fields have consistent styling!